src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
 
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  --catalog-columns: 1;
+  gap: 48px;
+  grid-template-columns: repeat(var(--catalog-columns),$card-width);
+  justify-content: center;
+
+  @media (min-width: 488px) {
+    --catalog-columns: 2;
+  }
+
+  @media (min-width: 768px) {
+    --catalog-columns: 3;
+  }
+
+  @media (min-width: 1024px) {
+    --catalog-columns: 4;
+  }
```

Comment:
Interesting solution, i like it !

Reply:
https://mateacademy.slack.com/archives/C03H0MLSFUJ/p1655716733610009

Reply:
I think i changed all bugs which you say me to fix?